### ISSUE TITLE
credentail .json is too short

### ISSUE THREAD

**Issue by zhengzong007** [eouia/MMM-Assistant#7]
Hi, 
I am following your instructions step by step to create credential json files, but it looks too short and doesnt have the key work "google assistant" and "path" in this file (400bytes size). could you think of any thing wrong with my work?
Best Regards,
Ta

**Comment by eouia** [comment]
Which step have you been stuck in? `credential json` size might be around 400bytes.
and contents could be similar with this;
```
{
  "installed":{
    "client_id":"blahblahsomething.apps.googleusercontent.com",
    "project_id":"my mirror project",
    "auth_uri":"https://accounts.google.com/o/oauth2/auth",
    "token_uri":"https://accounts.google.com/o/oauth2/token",
    "auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs",
    "client_secret":"blahblahsomething",
    "redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]
  }
}
```

**Comment by zhengzong007** [comment]
My json looks exactly like your sample...
I stuck at step 3-running "node google-auth.js" the terminal shows:
module.js:471
throw err;
^
Error:cannot find module 'google-assistant'
at function.module.........
at function.module.........
at....
...

do I need to create a "google-assistant" somewhere else?

**Comment by eouia** [comment]
I think you've failed in `Step 2. Install module`.
Check the directory exists.
```
cd ~/MagicMirror/modules/MMM-Assistant
ls ./node_modules/google-assistant
```
It could show some directories and files.

**Comment by zhengzong007** [comment]
I tried to run step 2 again, then run the ls ./node...
it seems google-assistant is not copied

ls: cannot access './node_modules/google-assistant': No such file or directory
so strange...

**Comment by eouia** [comment]
try this in your MMM-Assistant directory
```
npm install -s google-assistant
```

**Comment by eouia** [comment]
And Check this also.
```
cd ~/MagicMirror/modules/MMM-Assistant
ls package.json
```
See the `package.json` with any editor.
It should be same with `https://github.com/eouia/MMM-Assistant/blob/master/package.json`.

and then,
```
npm install
```
Which error message could you see?

**Comment by eouia** [comment]
> there is no ¨google-assistant¨ in MMM-Assistant...so is there any other instructions need to be done before we run these steps?
`google-assistant` is automatically installed during step 2 by `npm install` with `package.json`. So there is nothing to do by manual normally. 
But I don't know why you couldn't install that module.
Suspicious points are;
- You didn't `npm install` or at least be executed in wrong directory. 
- your `npm` has some problem.

**Comment by zhengzong007** [comment]
i think it was because the ¨npm install -s google-assistant¨ was missing from the instructions lol,  
after i ran this command the google-assistant file show up:

components  examples  index.js  lib  package.json  README.md

**Comment by eouia** [comment]
That is not missing. It is included in `package.json`, so it should be installed automatically.

**Comment by zhengzong007** [comment]
i was strictly following everything i think lol

**Comment by eouia** [comment]
Well, I don't know exactly what is wrong.
Anyway, there is possibility other modules are not installed properly, so you should check the modules in `package.json` and install them manually.

**Comment by zhengzong007** [comment]
ok, thanks a lot for your help tonight!!!! :)

**Comment by eouia** [comment]
So, can I close this issue? I'll close this. If you have another problem, create another issue.
Have a nice day. 👍

**Comment by zhengzong007** [comment]
yes, you can close this one now :) cheers!
